awesome_dashboard/controllers/controllers.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import random
+
+from odoo import http
+from odoo.http import request
+
+logger = logging.getLogger(__name__)
+
+class AwesomeDashboard(http.Controller):
+    @http.route('/awesome_dashboard/statistics', type='json', auth='user')
```

Comment:
no need to add the name of the module to the route, it should be self-explanatory 

```suggestion
    @http.route('/dashboard/statistics', type='json', auth='user')
```

Reply:
Solved